src/app/menu/menu.component.scss

```diff
@@ -0,0 +1,476 @@
+.format-button-login
+ {
+    width:auto;
+    color: white;
+    padding: 14px 20px;
+    margin: 8px 0;
+    border: none;
+    border-radius: 50px;
+    cursor: pointer;
+    width: 100%;
+}
+
+.format-menu
+{
+  margin: 0px;
+  padding: 0px;
+}
+
+.format-item-menu
+{
+  list-style: none;
+}
+
+.format-link
+{
+    color: #1F1F1F;
+    -webkit-transition: 0.3s;
+    -moz-transition: 0.3s;
+    -o-transition: 0.3s;
+    transition: 0.3s;
+    outline: medium none;
+}
+
+.format-link:focus 
+{
+    text-decoration: none;
+    outline: none;
+    color: #fff;
+}
+
+.format-link:focus, .format-link:hover,
+.portfolio-cat .format-link:hover,
+.footer -menu .format-item-menu .format-link:hover 
+{
+    text-decoration: none;
+}
+  
+.header-area 
+{
+    position: absolute;
+    left: 0;
+    right: 0;
+    width: 100%;
+    top: 0;
+    z-index: 9;
+    margin-top: 18px;
+}
+  
+@media (max-width: 767px) 
+{
+    .header-area 
+    {
+      padding-top: 0;
+    }
+  }
+  
+@media (min-width: 768px) and (max-width: 991px) 
+{
+    .header-area 
+    {
+        padding-top: 0;
+    }
+}
+  
+.header-area .main-header-area 
+{
+padding: 0 150px;
+}
+  
+@media (max-width: 767px) 
+{
+    .header-area .main-header-area 
+    {
+    padding: 10px 10px;
+    }
+}
+  
+@media (min-width: 768px) and (max-width: 991px) 
+{
+    .header-area .main-header-area 
+    {
+        padding: 10px 10px;
+    }
+}
+  
+@media (min-width: 992px) and (max-width: 1200px) 
+{
+    .header-area .main-header-area
+    {
+        padding: 0 20px;
+    }
+}
+  
+@media (min-width: 1200px) and (max-width: 1500px) 
+{
+    .container 
+    {
+        max-width: 100% !important;
+    }
+    .header-area .main-header-area 
+    {
+        padding: 0 10px;
+    }
+}
+  
+.header-area .main-header-area .logo-img 
+{
+    text-align: center;
+}
+  
+@media (max-width: 767px) 
+{
+    .header-area .main-header-area .logo-img 
+    {
+        text-align: left;
+    }
+}
+  
+@media (min-width: 768px) and (max-width: 991px) 
```

Comment:
Definition duplicated at 153, this also happens at others @media definitions. Please include everything from the same @media in the same place